package.json

```diff
@@ -34,13 +34,13 @@
   },
   "bin": {
     "cml": "bin/cml.js",
-    "cml-send-github-check": "bin/legacy.js",
-    "cml-send-comment": "bin/legacy.js",
-    "cml-publish": "bin/legacy.js",
-    "cml-tensorboard-dev": "bin/legacy.js",
-    "cml-runner": "bin/legacy.js",
-    "cml-cloud-runner-entrypoint": "bin/legacy.js",
-    "cml-pr": "bin/legacy.js"
+    "cml-send-github-check": "bin/legacy/link.js",
+    "cml-send-comment": "bin/legacy/link.js",
+    "cml-publish": "bin/legacy/link.js",
+    "cml-tensorboard-dev": "bin/legacy/link.js",
+    "cml-runner": "bin/legacy/link.js",
+    "cml-cloud-runner-entrypoint": "bin/legacy/link.js",
```

Comment:
Is this needed? I do not remember this command

Reply:
Yes, this is needed. It's the way we have of retaining compatibility with legacy `cml-command` invocations. When users install the package, `npm` will create symbolic links from e.g. `cml-runner` to `link.js` and the latter will decide what to run based on the link name.

Reply:
Speaking about
```
"cml-cloud-runner-entrypoint": "bin/legacy/link.js",
```

its that invoked as `cml-cloud-runner`? cml-runner works but not the other

Reply:
Ah! docker miight be?

Reply:
Whatever you did on https://github.com/iterative/cml/pull/487 😅

Reply:
Ok thats totally deprecated. Lets open a technical debt to remove that

Reply:
And break backwards compatibility? 🧛🏼 🧄 😉

Reply:
Sounds like something to add under the doubly epic #1090?

Reply:
😆 that not breaking anything. Indeed you and casper superseded that with your smart entrypoint later on

Reply:
```suggestion
```

Then, express removal it is.